libs/@local/graph/authorization/src/policies/mod.rs

```diff
@@ -135,6 +142,23 @@ impl Policy {
     pub(crate) fn try_from_cedar(
         policy: &ast::StaticPolicy,
     ) -> Result<Self, Report<InvalidPolicy>> {
+        // let resource_filter = match policy.condition().expr_kind() {
+        //     ast::ExprKind::Lit(ast::Literal::Bool(true)) => None,
+        //     ast::ExprKind::BinaryApp {
+        //         op: ast::BinaryOp::Contains,
+        //         arg1,
+        //         arg2,
+        //     } => {
+        //         let resource_filter =
+        //             if arg1.expr_kind() == ast::ExprKind::Lit(ast::Literal::Bool(true)) {
+        //                 Some(arg2)
+        //             } else {
+        //                 None
+        //             };
+        //     }
+        //     _ => return Err(InvalidPolicy::InvalidSyntax.into()),
+        // };
```

Comment:
Initial attempt to translate a filter from Cedar to our syntax. As we don't need it (yet) I commented out.